### DISCUSSION TITLE
Tip for using Primary, Secondary, Accent and Neutral.

### DISCUSSION THREAD

**Discussion by jjdevzinho** [saadeghi/daisyui#564] [General]
Hi, could anyone give me a hint on how much and where to use "Primary, Secondary, Accent and Neutral"?

Currently I'm only using Primary and Neutral

I use Neutral in Footer.
I use Primary in some Links, Badge and icons.

Are there right or more specific places to use each of these color classes? I'm really bad at designing and I don't want to pollute the website.

Website I'm building: https://www.fandesu.com/lista

Thanks for listening

**Comment by saadeghi** [comment]
Hey, Cool project! 👏
It really depends on the design and you might not need to use all colors.  
Some websites (like [this](https://www.adidas.com/) or [this](https://vercel.com/)) are just black and white and might not even need `primary`, `secondary`, etc.  
Many websites like [this](https://www.shopify.com/) would just need `neutral` and `primary`

**Reply by jjdevzinho** [reply]
Hi, thank you very much for the reply.

I'll just keep using the two classes then :)
